### ISSUE TITLE
Mutation json report wrong mutation span reported

### ISSUE THREAD

**Issue by Mobrockers** [stryker-mutator/stryker-net#346]
When the mutation is the first/only code in a line, the syntaxtree seems to not give us the correct span for the syntax node and/or we are misinterpreting something about the syntaxnode span.

See line with second--:
![image](https://user-images.githubusercontent.com/3595932/53042359-6702a780-3486-11e9-95d4-60bc66f760e0.png)

**Comment by Mobrockers** [comment]
This mutation is reported as the following:

```json
{
"id": 4,
"mutatorName": "PostDecrementExpression to PostIncrementExpression mutation",
"replacement": "\t\t\t\tsecond++",
"location": {
  "start": {
    "line": 11,
    "column": 1
  },
  "end": {
    "line": 11,
    "column": 13
  }
 }
}
```

Starting at column 1 is obviousy wrong. Might be due to the tabs.

**Comment by richardwerkman** [comment]
We could try to remove the trivia from each node before asking its span. The span seems to be correct, the trivia is just a part of the mutated node.

**Comment by Mobrockers** [comment]
Fixed by #373
